Directory.Build.props

```diff
@@ -23,12 +23,17 @@
     <Configuration Condition="'$(Configuration)'==''">Debug</Configuration>
     <Configurations>Debug;Release;Debug-netcoreapp3_1;Release-netcoreapp3_1;Debug-netfx;Release-netfx</Configurations>
     <Platform Condition="'$(Platform)'==''">AnyCPU</Platform>
-    <TargetArchitecture Condition="'$(TargetArchitecture)' == ''">x64</TargetArchitecture>
-    <NativeTargetArchitecture Condition="'$(NativeTargetArchitecture)' == ''">$(TargetArchitecture)</NativeTargetArchitecture>
+    <TargetArchitecture Condition="'$(TargetArchitecture)' == ''">$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)</TargetArchitecture>
+    <TargetArchitecture>$(TargetArchitecture.ToLower())</TargetArchitecture>
     <PlatformConfig>$(Platform).$(Configuration)</PlatformConfig>
     <StrongNameKeyId>Open</StrongNameKeyId>
   </PropertyGroup>
 
+  <ItemGroup>
+    <TargetsArm Include="LdaNative" />
```

Comment:
This item name is a bit odd. Maybe it should be `NativeAssembliesTargettingArm` or similar?

Reply:
We should also consider putting this next to the place in the `.targets` file that uses it. No real need to separate this into the `.props` file, when only 1 place in a target reads it.

Reply:
Makes sense. Renamed and moved to the `.targets` file.